### ISSUE TITLE
Cannot read property 'paginator' of null

### ISSUE THREAD

**Issue by AkostDev** [avocode/avocode#1934]
---
* **Version**: 2.21.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 133315
---
```
TypeError: Cannot read property 'paginator' of null
    at file:///C:/Users/Alex/AppData/Local/avocode/app-2.21.0/resources/app.asar/dist/avocode.min.js:1:2322063
    at file:///C:/Users/Alex/AppData/Local/avocode/app-2.21.0/resources/app.asar/dist/avocode.min.js:1:2303174
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by n1ckfedorov** [comment]
тоже самое..

**Comment by jakubzitny** [comment]
Hi @Woody174 and @N1ckFedorov, thanks for reporting it. We can see this in our bug tracking system. When exactly did it happen and how often do you see it please? We will work on a fix as soon as possible.

**Comment by jakubzitny** [comment]
Hi, this problem should be released in latest Avocode 2.22.0. Please update and let us know if it's still happening.
